src/Avalonia.Base/Platform/DefaultPlatformSettings.cs

```diff
@@ -52,5 +52,14 @@ protected void OnColorValuesChanged(PlatformColorValues colorValues)
         {
             ColorValuesChanged?.Invoke(this, colorValues);
         }
+
+        public virtual PlatformThemeVariant ThemeVariant => default;
+
+        public event EventHandler<EventArgs>? ThemeVariantChanged;
```

Comment:
I see how useful is API to get platform-specific theme variant. 
I think it's a good idea to add it, but with let's adjust it a bit, because `ThemeVariant` naming here is confusing.

Since other color specific parameters are already in `PlatformColorValues` class, it can be modified in this way:
```diff
    /// <summary>
-    /// System theme variant or mode.
+    /// Window theme variant or mode.
    /// </summary>
    public PlatformThemeVariant ThemeVariant { get; init; }

+    /// <summary>
+    /// System theme variant or mode.
+    /// </summary>
+    public PlatformThemeVariant SystemThemeVariant { get; init; }
```

Reply:
I can do this, but it's a bit unexpected to have window properties in the `IPlatformSettings` interface.

Also, each window can individually change its theme variant, so a value on this interface would only tell us the default setting for new windows.

Do you still want this change?